src/Tribe/I18n.php

```diff
@@ -150,14 +197,30 @@ protected function with_locale( $locale, callable $do, array $args = [] ) {
 	/**
 	 * Compiles the translations for a set of strings iterating on a set of domains.
 	 *
+	 * The 4th argument is a bitmask to control the compiled translations.
+	 * E.g. `$i18n->compile_translations( $strings, $domains, I18n::COMPILE_STRTOLOWER);` will only compile
+	 * translations of the strings in their `strtolower` versions.
+	 * Combine the flags using the usual PHP syntax: `I18n::COMPILE_INPUT | I18n::COMPILE_STRTOLOWER` to compile
+	 * only the translation of the string as input and in their lowercase version.
+	 *
 	 * @since 5.1.1
+	 * @since TBD   Add support for the $flags argument.
 	 *
 	 * @param array<string,array|string> $strings The set of strings to compile the translations for.
 	 * @param string|array<string>       $domains The domain(s) that should be used to compile the string translations.
+	 * @param int $flags                          An integer resulting from the combination of compilation flags;
+	 *                                            defaults to `static::COMPILE_ALL` to compile all versions of the
+	 *                                            translations.
+	 *                                            `static::COMPILE_INPUT` will compile the translation for the string,
+	 *                                            as input.
+	 *                                            `static::COMPILE_STRTOLOWER` will compile the translation for the
+	 *                                            string in its lowercase version.
+	 *                                            `static::COMPILE_UCFIRST` will compile the translation for the string
+	 *                                            in its title version.
 	 *
 	 * @return array<string|array> A map of the compiled string translations.
 	 */
-	 public function compile_translations( array $strings, $domains ) {
+	 public function compile_translations( array $strings, $domains, $flags = 7 ) {
```

Comment:
* :warning: Found precision alignment of 1 spaces.
* :no_entry_sign: Line indented incorrectly; expected 1 tabs, found 1

<sub>hash: 1a1ec1b2dd333d2603fb82c4794f9617</sub>